tests/foreman/cli/test_syncplan.py

```diff
@@ -119,6 +120,22 @@ def valid_name_interval_update_tests():
     ]
 
 
+@filtered_datapoint
+def valid_cron_expressions():
```

Comment:
Not blocker but this looks like duplicate code.. rather we can have this list in `constant.py`

Reply:
we can put it in datafactory.py module instead, but have to write a unittest

Reply:
Agree with @ldjebran , I see another use case for scap where we could use this, if we have it in datafactory

Reply:
Done!